guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md

```diff
@@ -174,9 +171,9 @@ class YourTestStep implements TestStepInterface
 {
     public function __construct
     (
-        // data that are required for this step along with other dependencies.
+        // data that is required for this step, along with other dependencies.
```

Comment:
data and other dependencies required for this step

Reply:
Right.